src/components/Person/Person.jsx

```diff
@@ -1 +1,24 @@
 // export function Person({ person }) {}
+import React from 'react';
+
+export function Person({ name, age, sex, isMarried, partnerName }) {
+  const married = `My ${(sex === 'f') ? `husband's` : ` wife's`} name is ${partnerName}`;
+  const notmarried = `I am not married`;
```

Comment:
Names could be better - they sound near the boolean value
better smth like `partnerInfo` and `notMarriedStatus` IMHO